muffnn/mlp/mlp_classifier.py

```diff
@@ -262,3 +262,8 @@ def __getstate__(self):
             state['n_classes_'] = self.n_classes_
 
         return state
+
+    def score(self, X, y):
+        accuracy = np.array(y) == self.predict(X)
+        accuracy = accuracy[np.array(y) != -1].mean(axis=0).mean()
```

Comment:
Ahh yes. I remember now. I think rows, you think columns, and we are doing none of those. OK. Let's merge as is, or go for columns. I will leave the decision up to you, but merging as is would be fine by me. 

Thanks for bearing with my poor memory!